src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -2003,6 +2003,44 @@
      * The {@code continue} keyword.  The first child is an
      * optional identifier and the last child is a semicolon.
      *
+     * <p>For example:</p>
+     * <pre>
+     * for (int i = 0; i < 5; i++) {
+     *     continue;
+     * }
```

Comment:
for simplicity lets change example to 
```
for (;;) {
    continue;
}
```

do not forget to update AST tree

Reply:
Done.